test/coverlet.core.tests/InstrumenterHelper.cs

```diff
@@ -191,7 +191,9 @@ public static CoverageResult GetCoverageResult(string filePath)
             using (var result = new FileStream(filePath, FileMode.Open))
             {
                 CoveragePrepareResult coveragePrepareResultLoaded = CoveragePrepareResult.Deserialize(result);
-                Coverage coverage = new Coverage(coveragePrepareResultLoaded, new Mock<ILogger>().Object, new InstrumentationHelper(new ProcessExitHandler(), new RetryHelper(), new FileSystem()));
+                var fileSystem = new Mock<IFileSystem>();
```

Comment:
Revert changes to this file...we need real FileSystem, this is an helper to test out of process instrumentation and must behave like "production".

Reply:
Done